### DISCUSSION TITLE
Autofee

### DISCUSSION THREAD

**Discussion by ElBiasto** [cryptosharks131/lndg#204] [Q&A]
I have Autofee turned on for all of my channels. When I checked the Logs today. I found something that looks strange to me: 
![image](https://user-images.githubusercontent.com/36233370/217026914-6b0fc206-cab0-4fad-91c4-47766f3b4b32.png)
I had a channel with some activity when fees have been at 5ppm. Why would Autofee reduce the fees to 1 in this case? Shouldn't it raise the fee in order to earn more if a channel is used frequently?

**Comment by cryptosharks131** [comment]
It's hard to say what's going on here by only looking at this screen shot and without knowing some details of the channel but in general, AutoFees will try to reduce when outbound liquidity is above 25% and the assisted ratio is strong or the flow is weak.
